api/v1alpha1/addon_types.go

```diff
@@ -174,11 +174,23 @@ type StorageClassConfig struct {
 	AllowExpansion bool `json:"allowExpansion,omitempty"`
 }
 
+type CSICredentials struct {
```

Comment:
I think the examples need to change too https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/tree/main/examples/capi-quick-start, and possible some snippets in docs

Reply:
Good catch, I'll update that

Reply:
Fixed in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/663/commits/18dbf4def55e390325a1f056a3abd9c40aeaea50